docs/source/starter/lightning_lite.rst

```diff
@@ -3,15 +3,14 @@ LightningLite - Stepping Stone to Lightning
 ###########################################
 
 
+:class:`~pytorch_lightning.lite.LightningLite` enables pure PyTorch users to scale their existing code
+on any kind of device while retaining full control over their own loops and optimization logic.
+
 .. image:: https://pl-public-data.s3.amazonaws.com/docs/static/images/lite/lightning_lite.gif
-    :alt: Animation showing how to convert a standard training loop to a Lightning loop
-    :width: 600px
+    :alt: Animation showing how to convert your PyTorch code to LightningLite.
+    :width: 400
```

Comment:
this looks a bit too small, the font is smaller than the text. how about 500?

Reply:
@awaelchli Any idea why it doesn't get centered ?

Reply:
I couldn't find a way. It's possible that a style in the template prevents us from using any center options. Personally I don't find it that important to center it. If we can center it somehow, it would probably be better to make the background fully white :)